src/main/java/pl/gov/coi/pomocua/ads/work/WorkResource.java

```diff
@@ -0,0 +1,33 @@
+package pl.gov.coi.pomocua.ads.work;
+
+import lombok.AllArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.Optional;
+import java.util.UUID;
+
+@RestController
+@AllArgsConstructor
+public class WorkResource {
+    private final WorkOfferRepository repository;
+
+    @PostMapping("/api/secure/offer/work")
```

Comment:
What do you think about having offer in plural? I am also wondering if we plan design API in a way that there will be different typed of offers? The `/work` part of endpoint looks like a filter of offers which would be a param (ex. `?type=work`).
What do you think about changing it into:
`/api/job-offers/`
or
`/api/offers?type=work`?

Reply:
I like `job-offers`!

Reply:
Now `/api/jobs` came to my mind, WDYT?

Reply:
Sounds even better :)